### ISSUE TITLE
Feature request: Request new mentor

### ISSUE THREAD

**Issue by codeholic** [exercism/exercism#4758]
Some mentors have got a too formalistic approach and do not provide valuable feedback, even when explicitly asked for it. Unfortunately these mentors are at the same time among the most active ones (at least on certain tracks), which totally disrupts the whole learning process and frustrates a lot.

As a student, I want to be able to request a new mentor, when I'm not satisfied with their performance.

**Comment by iHiD** [comment]
Hello. Sorry you've not had a great experience. Could you provide some sample solution IDs pls? It you don't want to do that publicly, you can email them to jeremy@exercism.io - that will help us understand the problem more clearly. Thanks.

**Comment by jramdohr** [comment]
Hello @codeholic,

my mentors account was deleted and it would be great if there would be an option to request a new mentor to finish and continue with the exercise. It seems like I am currently stuck in limbo.

Thanks!

**Comment by iHiD** [comment]
Hi @jramdohr - if you open a new issue with your solution url and specify the track, someone will look for you.

**Comment by jasper2virtual** [comment]
I have the same experience now, my solution is getting the job done and meet the requirement on instructions and hints, and very simple and readable. But my mentor just keep asking for me to rewrite it, in his own complex style.

Anyway, I understood and accept his suggestion to rewrite the code. As a student, we must willing to open our mind to accept mentor opinions, we have less coding experience and skill then mentor, if we like to learn, first we must empty our cup first, no matter bad or good, we digest it first.

Exercism is a free learning service, all the mentor are not paid to work. I think as a student, we already have a choice, to choosing the Practice Mode, if you are self confident on your skill. 

Maybe, add a internal function that, mentor can inspect each other mentor comment, if the solution re-submit too much.

**Comment by kotp** [comment]
@jasper2virtual If you feel like you have a working relationship with your mentor you can ask for what you want from them.  For example:

"Can we approve this iteration, I (do or do not) want to continuing exploring the feedback you are giving, but want to be able to also work on the next exercises."

"No offense (if not offended, of course) but can I request that this is not approved, but I am placed back in the queue for another mentor?  I (have or have not) enjoyed the mentoring, but feel like another mentor may be better suited for me."

Hopefully you feel like you can talk with the mentor there.  We are all human, and we know that we don't always mesh with everyine.  If you feel like you can not talk with your mentor, and feel uncomfortable having that communication with them, then please do "you can email them to jeremy@exercism.io" as stated by @iHiD above.

Of course for anything tips the abuse scale, e-mail abuse@exercism.io so that the first staff member that reads it can direct it appropriately and get that ball started.

**Comment by iHiD** [comment]
Thanks for the post. So FYI, this all changes in v3 coming soon, where you can leave conversations as a student at any point, and your progress is no longer blocked by mentors.

**Comment by jasper2virtual** [comment]
Keep empty your cup. At the end my mentor guide me to a solution which i never thought before.

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂
